sklearn/tree/tests/test_tree.py

```diff
@@ -1992,3 +1992,60 @@ def test_prune_tree_raises_negative_ccp_alpha():
     with pytest.raises(ValueError, match=msg):
         clf.set_params(ccp_alpha=-1.0)
         clf._prune_tree()
+
+
+def test_multilabel_invariance():
```

Comment:
I think this test is excessive. It is impossible and inappropriate to try test every variant of tree building is handled the same for the two formats. A basic check that they are handled the and should be reasonably sufficient if we know that they share validation code.

Reply:
ack